packages/core/http/core-http-server-internal/src/csp/config.ts

```diff
@@ -83,6 +83,13 @@ const configSchema = schema.object(
     report_to: schema.arrayOf(schema.string(), {
       defaultValue: [],
     }),
+    report_only: schema.maybe(
+      schema.object({
+        form_action: schema.arrayOf(schema.string(), {
+          defaultValue: [],
+        }),
```

Comment:
__note:__ looking at the logic for merging default values with the values from the config in `/core-http-server-internal/src/csp/csp_directives.ts`, it seems we only allow adding directives (e.g., it's not possible to remove `'self'` from `form_action` with `form_action: []`). It's fine, but in this case, it would probably make sense to validate the values with `validate: getDirectiveValidator({ allowNone: false, allowNonce: false })` like we do for other directives so that we don't end up with non-compatible values (e.g., `'self' 'none'`). 

What do you think?

Reply:
Good catch, added validation